nomt/src/seek.rs

```diff
@@ -77,6 +80,10 @@ impl SeekRequest {
             panic!("seek past end")
         };
 
+        self.continuations += 1;
+
+        assert!(self.continuations < 10);
```

Comment:
Why is there this limit? Why is `continue_seek` only callable a maximum of 10 times? Or am I just looking at it wrong?

The trie is made up of a maximum of 42 pages (height), so if continue_seek just walks through one page at a time, isn't 10 too small a limit?

Reply:
removed in #307

Reply:
yeah it's actually a debugging thing I accidentally left in :see_no_evil: